code/game/objects/items/RPD.dm

```diff
@@ -252,9 +254,56 @@ GLOBAL_LIST_INIT(fluid_duct_recipes, list(
 	spark_system = null
 	return ..()
 
+/obj/item/pipe_dispenser/examine(mob/user)
+	. = ..()
+	. += "You can scroll your mouse wheel to change the piping layer."
+
+/obj/item/pipe_dispenser/equipped(mob/user, slot, initial)
+	. = ..()
+	if(slot == ITEM_SLOT_HANDS)
+		RegisterSignal(user, COMSIG_MOUSE_SCROLL_ON, .proc/mouse_wheeled)
```

Comment:
You should probably register a signal for when the dispenser changes loc (ATOM_MOVE) and then unregister the scroll if the loc is no longer the player's hands because a lot of things forcemove items. Also unregister the movement signal.